packs/feats/energy-blessed.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "ancestry",
         "description": {
-            "value": "<p>Energy runs through your blood more powerfully. When you use Energy Emanation, you can create an emanation of 5 feet, 10 feet, or 15 feet. The damage increases to 6d6 plus an additional 1d6 for every level you have above 9th, instead of 1d6 plus an additional 1d6 for every 2 levels above 1st.</p>"
+            "value": "<p>Energy runs through your blood more powerfully. When you use Energy Emanation, you can create an @Template[type:emanation|distance:5]{emanation of 5 feet}, @Template[type:emanation|distance:10]{10 feet}, or @Template[type:emanation|distance:15]{15 feet}. The damage increases to @Damage[(6+max(0,@actor.level - 9))d6[@actor.flags.pf2e.elementalHeartDwarf.damageType]|traits:area-damage], instead of 1d6 plus an additional 1d6 for every 2 levels above 1st.</p>"
```

Comment:
```suggestion
            "value": "<p>Energy runs through your blood more powerfully. When you use Energy Emanation, you can create an emanation of @Template[type:emanation|distance:5]{5 feet}, @Template[type:emanation|distance:10]{10 feet}, or @Template[type:emanation|distance:15]{15 feet}. The damage increases to @Damage[(6+max(0,@actor.level - 9))d6[@actor.flags.pf2e.elementalHeartDwarf.damageType]|traits:area-damage], instead of 1d6 plus an additional 1d6 for every 2 levels above 1st.</p>"
```